index.js

```diff
@@ -635,6 +635,22 @@
   //. false
   //. ```
 
+  //# Either#extend :: Either a b ~> (b -> b) -> Either a b
+  //.
+  //. Takes a function and returns `this` if `this` is a Left; otherwise it
+  //. returns a Right whose value is the result of applying the function to
+  //. this Right's value. `Either#extend` is a restricted form of
+  //. [`Either#map`][1]: the function provided must return a value of the
```

Comment:
This should link to the appropriate section of the documentation. I suggest

    [`Either#map`](#Either.prototype.map)

since we're linking to a section in the same document.